src/SuperSocket.Connection/PipeConnectionBase.cs

```diff
@@ -97,7 +97,8 @@ private async ValueTask HandleClosing()
         {
             try
             {
-                await _pipeTask.ConfigureAwait(false);
+                if (_pipeTask != null)
```

Comment:
add comment for the scenario _pipeTask = null

Reply:
You can put this change into a separated PR.

Reply:
ok